Reworded "list of objects" to remove any ambiguity

DanKaplanSES opened the pull request:
### Description, Motivation, and related issues

Fixes #30727

DanKaplanSES committed:
Reworded "list of objects" to remove any ambiguity

- Context: #30727

teoli2003 commented:
@DanKaplanSES A technicality: I've added the word _Fixes_ in your description: that way when this PR is merged, GitHub will automatically close the issue (that I reopen).

That's the standard way of doing it.

DanKaplanSES commented:
> @DanKaplanSES A technicality: I've added the word _Fixes_ in your description: that way when this PR is merged, GitHub will automatically close the issue (that I reopen).
> 
> That's the standard way of doing it.

Oh! Thank you. I had know idea.

DanKaplanSES committed:
Removed and incorrect comma

- I'm not 100% sure of this change, but I'm *pretty* confident this comma should be removed.

DanKaplanSES committed:
After further research, I think the comma was required after all

wbamberg commented on files/en-us/web/api/console/index.md:
```diff
@@ -76,7 +76,7 @@ This page documents the [Methods](#methods) available on the `console` object an
 The console's most frequently used feature is logging text and other data. There are several categories of output you can generate using the {{domxref("console/log_static", "console.log()")}}, {{domxref("console/info_static", "console.info()")}}, {{domxref("console/warn_static", "console.warn()")}}, {{domxref("console/error_static", "console.error()")}}, or {{domxref("console/debug_static", "cons ...
 
-There are two ways to use each of the output methods; you can pass in a list of objects whose string representations get concatenated into one string, then output to the console, or you can pass in a string containing zero or more substitution strings followed by a list of objects to replace them.
+There are two ways to use each of the output methods; you can pass in a variable number of arguments whose string representations get concatenated into one string, then output to the console, or you can pass in a string containing zero or more substitution strings followed by a variable number of arguments to replace them.
```
Do you think it is worth bulleting this to make it easier to read?

e.g.

```suggestion
There are two ways to use each of the output methods:
- Pass in a variable number of arguments whose string representations get concatenated into one string, then output to the console.
- Pass in a string containing zero or more substitution strings followed by a variable number of arguments to replace them.
```

just a suggestion, if you think it is worth doing.

DanKaplanSES replied:
Yep, I considered that. I think that's a good idea.

DanKaplanSES committed:
Update files/en-us/web/api/console/index.md

- Use bullet points when describing each output method.

Co-authored-by: wbamberg <will@bootbonnet.ca>

wbamberg committed:
Update files/en-us/web/api/console/index.md

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

wbamberg reviewed (approved):
👍 thank you @DanKaplanSES !
